_posts/randy/2022-09-10-tech-juggling-reality

```diff
@@ -0,0 +1,171 @@
+
+## PRODUCE A WHITE PAPER
+
+If warranted, a technical WHITE PAPER can go long ways to help the market and your customers visially understand the PROBLEM at hand, and the SOLUTION you are handing them.
+
+Assume there will be RESISTANCE to whatever it is you are starting out to do. Work with your entire team to craft a message that portrays a deliverable win for all. Explain technical terms in layman words, so all can understand.
+
+## PRACTICE & PERFECT THE PITCH
+
+Any brand or service should be able to be summarized adequately in a couple of sentences. Study this part hard; not hardly. Make sure everyone in the company can enunciate the reason for being.
+
+Put some creative effort into promotional materials. Physical swag is a waste for most technology plays – put money into print and video, and combine with online campaigns. Polish the show to a shine – make everyone look good.
+
+## NETWORKING NETS WORK
+
+Getting your backside out into the real world will always be a favored method for exposing your business. Key executives, sales and technical teams all need to rub shoulders with other smart people – spark off some positive electricity.
+
+If there is a relevant gathering of industry personnel, calculate the long term benefits from such short term activities, and go for it. Get your brand off the couch:
+
+    ATTEND TRADE SHOWS, EXHIBITIONS, CONVENTIONS 
+
+    JOIN INDUSTRY ASSOCIATIONS 
+
+    VOLUNTEER YOUR EXPERIENCE 
+
+    HOLD OFFICE OPEN HOUSES 
+
+    SPONSOR ACTIVITIES & EVENTS 
+
+## BE ON THE HUNT FOR TOMORROW'S LUNCH
+
+At every stage of business, you can bet someone out there is building a better mousetrap or bigger circus tent – and coming for your pieces of cheese. Being successful in business requires constant efforts to soap, scrub and polish up ALL viable plates and keep them spinning revenue.
+
+Keep your senses aware, and always look for the next ball to drop. It may be a chance to clean up in the market.
+
+LATHER.
+
+RINSE.
+
+REPEAT...
+
+...
+
+Randy Berg
+
+xyz digital inc.
+
+...
+
+## RECOMMENDED READING
+
+[Technology Business Ideas - Profitable Adventure](https://www.profitableventure.com/technology-business-ideas)
+
+[3 Ways to Prepare Your Business for the Inevitable Quantum Revolution – Entrepreneur](https://www.entrepreneur.com/article/433374)
+
+[Business Technology Trends - Rasmussen U](https://www.rasmussen.edu/degrees/business/blog/business-technology-trends)
+
+[Create a Tech Startup in 12 Steps](https://onlinedegrees.sandiego.edu/how-to-create-a-tech-startup-company-in-12-steps-2)
+
+[Launching Tech Startups - Complete Guide](https://www.neoito.com/blog/launching-tech-startup-complete-guide)
+
+[Industrial Digitalization - Hitachi](https://social-innovation.hitachi/en-us/think-ahead/manufacturing/industrial-digitalization-for-smart-manufacturing) {NOTE – extensive use of buzz-phrases}
```

Comment:
I can't comment on the wav file below.... but are you hoping to include in the article?  I can show you the syntax for that, but we should include as mp3 not wav.

Reply:
i could not find the devpage where you had an audio file to compare code - will find a way to convert to mp3 and hold my nose same time.

Reply:
I used to have an example on one of my articles.... but ended up changing to a Kaizen embed.  Which reminds me, that that's what we should plan to do for yours as well.

Until then, you can use:

```
<audio controls="controls">
  <source type="audio/mp3" src="{{"/assets/audio/impossible-quote.mp3" | relative_url}}"></source>
  <p>Your browser does not support the audio element.</p>
</audio>
```

Just make sure to replace `impossible-quote.mp3` with the name of your file and put it in the `audio` folder under the assets folder.